### PR TITLE
Log domain XMLs on test failures

### PR DESCRIPTION
**Pull request by rmohr**
**What this PR does / why we need it**:

Log domain XMLs on test failures.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/reporter/kubernetes.go [hunk 1]
```diff
@@ -76,9 +78,37 @@ func (r *KubernetesReporter) SpecDidComplete(specSummary *types.SpecSummary) {
...
+
+	f, err := os.OpenFile(filepath.Join(r.artifactsDir, fmt.Sprintf("%d_domains.log", r.failureCount)),
+		os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to open the file: %v", err)
+		return
+	}
+	defer f.Close()
+
+	vmis, err := virtCli.VirtualMachineInstance(v1.NamespaceAll).List(&v12.ListOptions{})
```
Comment (fromanirh) [inline tests/reporter/kubernetes.go]: v-twelve? typo I believe

Comment (rmohr) [reply tests/reporter/kubernetes.go]: Ah that is autocomplete on import. I am sometimes lazy with renaming the import aliases to something useful. Will address it.

**Comment by slintes** [comment]
nice!

/lgtm
/approve

File: tests/reporter/kubernetes.go (hunk 1 above)
Comment (slintes) [reply tests/reporter/kubernetes.go]: this was not introduced with this PR, not worth waiting for a improvement. Will merge.

**Pull request merged**